WcaOnRails/app/views/incidents/_competition_tag.html.erb

```diff
@@ -0,0 +1,4 @@
+<% comp ||= Competition.new %>
```

Comment:
What is this here for?

Reply:
I have no clue, I guess I was worried about a user not passing a valid comp...